### PR TITLE
Creation of metric summary file

### PR DESCRIPTION
**Pull request by obto**
To help analysts get a quick overview of all the metrics we are tracking we will start providing summaries of each metric contained in each custom metrics file

### CODE REVIEW

File: custom_metrics/metric-summary.md [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+    "name": 8,
+    "content": 15,
+    "property": 7
+  }
+}
+```
+
+## 10.5
+
+*DEPRECATED*
```
Comment (rviscomi) [inline custom_metrics/metric-summary.md:118]: How should we think about deprecated metrics? Are there always better replacements somewhere else (if so where) or should we delete them?

**Review by rviscomi — COMMENTED** [review]
This is awesome, thanks for documenting all of this! Just one question about deprecated metrics.

File: custom_metrics/metric-summary.md (hunk 1 above)
Comment (obto) [reply custom_metrics/metric-summary.md:118]: Yes, both of those deprecated metrics are in the SEO chapters metrics. Once the wpt_bodies.js file is written in here, we can link that the new version can be found there.

Comment (rviscomi) [reply custom_metrics/metric-summary.md:118]: Ok SGTM. Could you add a link to that JS file so if someone is looking to query this data and finds this documentation first, they know where to look instead? Once we have docs for the other file it'll be easier to link together but we can link to the source JS itself in the mean time.

Comment (obto) [reply custom_metrics/metric-summary.md]: Done

**Review by rviscomi — APPROVED** [review]

**Pull request merged**